app/src/main/java/com/codesoom/assignment/SuperHttpHandler.java

```diff
@@ -0,0 +1,38 @@
+package com.codesoom.assignment;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.util.stream.Collectors;
+
+public class SuperHttpHandler implements HttpHandler {
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        // 1. Method - GET, POST, PUT/PATCH, DELETE, ...
+        // 2. Path - "/", "/tasks", "/tasks/1", ...
+        // 3. Headers, Body(Content)
+
+        String method = exchange.getRequestMethod();
+        URI requestURI = exchange.getRequestURI();
+        String path = requestURI.getPath();
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        if(path.startsWith("/tasks")){
+            TaskHandler taskHandler = new TaskHandler();
+            taskHandler.handler(method, path, body, exchange);
+        }else {
```

Comment:
사용하고 계신 IDE에서 포매터를 사용하는 방법을 조사해 보세요. 단축키로 실행할 수도 있고, 파일을 저장할 때마다 자동으로 돌아가게 만들 수도 있습니다. 조사를 완료하면 그 때부터는 이런 종류의 스페이스 문제는 신경쓰지 않게 될 거에요.

```suggestion
        if (path.startsWith("/tasks")) {
            TaskHandler taskHandler = new TaskHandler();
            taskHandler.handler(method, path, body, exchange);
        } else {
```

Reply:
[google-java-format 사용법](https://johngrib.github.io/wiki/google-java-format/) 과
save action 플러그인을 조합하여 적용하였습니다.